Cargo.toml

```diff
@@ -12,7 +12,7 @@ license = "MIT OR Apache-2.0"
 [dependencies]
 num = "0.1"
 num-derive = "0.1"
-parser-c-macro = "0.1.0"
+parser-c-macro = { path = "parser-c-macro" }
```

Comment:
If we bump `parser-c-macro/Cargo.toml` to 0.2.0 (just in case) and change this line, I believe we can both use a local copy and retain the ability to publish without modifications:

```
parser-c-macro = { path = "parser-c-macro", version = "0.2.0" }
```